ci-operator/config/openshift-priv/ovn-kubernetes/openshift-priv-ovn-kubernetes-release-4.16.yaml

```diff
@@ -501,8 +501,8 @@ tests:
       COMPUTE_NODE_TYPE: m5.2xlarge
       CONTROL_PLANE_INSTANCE_TYPE: r5.12xlarge
       EXTRA_FLAGS: --churn-duration=20m --service-latency --check-health=false --gc=false
+      METRICS_PROFILE: metrics.yml,metrics-report.yml
```

Comment:
Same. 750node should have aggregated. (does not need to be addressed in this PR)